moduleroot/.github/workflows/acceptance.yml.erb

```diff
@@ -0,0 +1,54 @@
+<%- if Dir[File.join(@metadata[:workdir], 'spec', 'acceptance', '**', '*_spec.rb')].any? -%>
```

Comment:
it feels weird to have that `if` here instead of modulesync dynamically dropping that file in place if that's true

Reply:
I couldn't quite figure out how to tell modulesync to not store the file and this at least prevents running it. In modules you can set `delete: true` in `.sync.yml` to remove it. However, only puppet-katello lacks acceptance tests (which is an issue on its own IMHO). After that module adds it, we can probably get rid of this.